adam/learner/attributes.py

```diff
@@ -70,45 +70,48 @@ def _candidate_templates(
+                        and not language_concept_alignment.token_index_is_aligned(
+                            preceding_token_index
+                        )
+                    ):
+                        ret.append(
+                            SurfaceTemplateBoundToSemanticNodes(
+                                language_concept_alignment.to_surface_template(
+                                    {object_node: SLOT1},
+                                    restrict_to_span=Span(
+                                        preceding_token_index, span_for_object.end
+                                    ),
+                                    language_mode=self._language_mode,
                                 ),
-                                language_mode=self._language_mode,
-                            ),
-                            {SLOT1: object_node},
+                                {SLOT1: object_node},
+                            )
                         )
-                    )
-                following_token_index = span_for_object.end + 1
-                if following_token_index < len(
-                    language_concept_alignment.language.as_token_sequence()
-                ) and not language_concept_alignment.token_index_is_aligned(
-                    following_token_index
-                ):
-                    ret.append(
-                        SurfaceTemplateBoundToSemanticNodes(
-                            language_concept_alignment.to_surface_template(
-                                {object_node: SLOT1},
-                                restrict_to_span=Span(
-                                    span_for_object.start, following_token_index
+                    following_token_index = span_for_object.end + 1
+                    if following_token_index < len(
+                        language_concept_alignment.language.as_token_sequence()
+                    ) and not language_concept_alignment.token_index_is_aligned(
+                        following_token_index
+                    ):
+                        ret.append(
+                            SurfaceTemplateBoundToSemanticNodes(
+                                language_concept_alignment.to_surface_template(
+                                    {object_node: SLOT1},
+                                    restrict_to_span=Span(
+                                        span_for_object.start, following_token_index
+                                    ),
+                                    language_mode=self._language_mode,
                                 ),
-                                language_mode=self._language_mode,
-                            ),
-                            {SLOT1: object_node},
+                                {SLOT1: object_node},
+                            )
                         )
-                    )
+                except RuntimeError:
+                    continue
```

Comment:
Why are we ignoring a runtime error here?

Reply:
Added a comment!